tests/testthat/test-filter.R

```diff
@@ -0,0 +1,100 @@
+# TODO: implement these functions in filters.R and use them in this test
+include <- function(...) {}
+exclude <- function(...) {}
+lessthan <- function(...) {}
+morethan <- function(...) {}
+interval <- function(...) {}
+
+# TODO: implement filters, check number of samples after filtering the test data and fill expected_samples
```

Comment:
In my mind, you should manually check the value of the expected samples and fill it in before developing the entire function to check all works properly, not after; If you create a mistake in the function, you will not ever know about it otherwise.

Reply:
fixed